cmd/repo-init/main.go

```diff
@@ -570,15 +579,29 @@ func generateCIOperatorConfig(config initConfig, originConfig *api.PromotionConf
 	}
 
 	for _, test := range config.CustomE2E {
-		generated.Configuration.Tests = append(generated.Configuration.Tests, api.TestStepConfiguration{
-			As:       test.As,
-			Commands: test.Command,
-			OpenshiftInstallerSrcClusterTestConfiguration: &api.OpenshiftInstallerSrcClusterTestConfiguration{
-				ClusterTestConfiguration: api.ClusterTestConfiguration{
-					ClusterProfile: test.Profile,
+		t := api.TestStepConfiguration{
+			As: fmt.Sprintf("test-e2e-%s", test.Profile),
```

Comment:
We should not prefix with `test-` and ideally we would make these agnostic out of the box, so not including the profile

Reply:
We already hardcode the name in `PromotesWithOpenShift` case with `e2e-aws`. There is a chance to end up with a duplication here. I will try to find another workaround though